libs/filaflat/src/BlobDictionary.h

```diff
@@ -29,8 +29,14 @@ class BlobDictionary {
     BlobDictionary() = default;
     ~BlobDictionary() = default;
 
+    using Blob = std::vector<uint8_t>;
+
     inline void addBlob(const char* blob, size_t len) noexcept {
-        mBlobs.push_back({ blob, len });
+        mBlobs.emplace_back(Blob(blob, blob + len));
+    }
+
+    inline void addBlob(Blob&& blob) noexcept {
+        mBlobs.emplace_back(blob);
```

Comment:
To avoid a `Blob` copy-construction inside `mBlobs` , I think you can do
```
mBlobs.emplace_back(std::move(blob));
```

Reply:
`blob` is an lvalue reference to an rvalue. In order for `emplace_back` to treat `blob` as an rvalue, `std::move` must be applied again.

https://stackoverflow.com/questions/28483250/rvalue-reference-is-treated-as-an-lvalue

Reply:
I ❤C++!  Fixed.  Thanks again.  :)